cmd/time-box/start.go

```
@@ -0,0 +1,51 @@
+	cycles   int
+)
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Start the Pomodoro timer",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := context.WithCancel(cmd.Context())
+		defer cancel()
+
+		phs := []timer.Phase{
+			{"Work", time.Duration(workMin) * time.Minute},
```

golangci-lint (1.64.8)

composites: github.com/K0201N/time-box/internal/timer.Phase struct literal uses unkeyed fields

(govet)